aws-ts-static-website/index.ts

```diff
@@ -66,7 +63,7 @@ crawlDirectory(
                 key: relativeFilePath,
 
                 acl: "public-read",
```

Comment:
```suggestion
```

With the new default settings ACLs are disabled so we can't/shouldn't set acls on the object itself.